Libplanet/Blockchain/BlockChain.cs

```diff
@@ -550,9 +550,11 @@ bool renderActions
 
                 string ns = Id.ToString();
                 HashDigest<SHA256>? tip = Store.IndexBlockHash(ns, -1);
+                IEnumerable<Transaction<T>> transactions = block.Transactions.
+                    OrderBy(tx => tx.Nonce).ToImmutableArray();
```

Comment:
Is this intended?

```suggestion
                    OrderBy(tx => tx.Id).ToImmutableArray();
```

Reply:
No, I intended to order by its `Nonce`. because `Transactions` of `Block<T>` are ordered by their `Id`, but while validating `Nonce` of `Transactions` it needs to be ordered by its `Nonce`